layers/b2qt/recipes-core/images/core-image-pelux-qtauto-neptune.bb

```diff
@@ -8,5 +8,5 @@ DESCRIPTION = "Reference PELUX image with QtAuto frontend"
 inherit core-image-pelux-qtauto
 inherit populate_sdk_qt5
 
-# This image uses neptune as the reference UI
-IMAGE_INSTALL += " neptune-ui "
+# This image uses neptune as the reference UI, which is one of the addons in QtAS packagegroup
```

Comment:
QtAS was discarded as a shorthand for Qt Automotive by the TQC marketing, we should use `Qt Auto` instead. The same goes for the commit message.